extensions/guacamole-auth-sso/modules/guacamole-auth-sso-saml/src/main/java/org/apache/guacamole/auth/saml/acs/SAMLService.java

```diff
@@ -98,6 +116,9 @@ public URI createRequest() throws GuacamoleException {
             throw new GuacamoleServerException("SAML IdP redirect could not "
                     + "be generated due to an error in the URI syntax: "
                     + e.getMessage());
+        } catch (SettingsException e) {
```

Comment:
Please use the same style as already established here (don't cuddle the `catch`).

Reply:
Sure